mathics/format/json.py

```diff
@@ -208,3 +210,23 @@ def sphere_3d_box(self) -> list:
 
 
 add_conversion_fn(Sphere3DBox, sphere_3d_box)
+
+
+def uniform_polyhedron_3d_box(self) -> list:
+    face_color = self.face_color
+    if face_color is not None:
+        face_color = face_color.to_js()
+    data = [
+        {
+            "type": "uniformPolyhedron",
+            "color": face_color[:3],
+            "faceColor": face_color,
```

Comment:
This won't be used in mathics-threejs-backend.

Reply:
I am not sure which field exactly you mean, but all of these are used. I tried setting facecolor and that works. 

BTW, since faceColor is there,  wiki entry for UniformPolyhedra in mathics-threejs-backend probably should be updated to include this field.

Reply:
`faceColor` should be replaced in all places by `color`, it works because of the translation layer.